x/cdp/client/cli/query.go

```diff
@@ -225,3 +227,55 @@ func QueryParamsCmd(queryRoute string, cdc *codec.Codec) *cobra.Command {
 		},
 	}
 }
+
+func QueryTotalSupply(queryRoute string, cdc *codec.Codec) *cobra.Command {
```

Comment:
you can already do this with the supply module 
`kavacli q supply total usdx`

Reply:
👍 Will remove the CDP specific query